STYLE.md

```diff
@@ -0,0 +1,126 @@
+# Style Guide
+
+Some of the code will disagree with this guide. Older code is less likely to be correct. Use your best judgement.
+
+When you come across an edge case that isn't referenced in this guide, please add it.
+
+
+## Comments
+
+```asm
+; Use spaces for indentation and alignment.
+
+; Comments lead with spaces after the semicolon.
+
+; 80 char soft limit. This isn't enforced, but you should try to keep lines from being any longer.
+; rgbasm doesn't have newline escapes so there's no avoiding exceeding the limit for longer macros.
+
+; Code comments should be wrapped to whatever
+; is most readable rather than what's closest
+; to the 80 char limit.
+
+; Comments should go above the code they're describing, not below, and be aligned with the code.
+
+; Functions are commented with unindented comments
+SomeFunction::
+    ; Code is commented with an indentation
+    ld  a, [hl]
+    add b
+    ld  [hl], a
+    ret
+
+    ; Avoid comments on the same line as instructions
+    cp 10
+    jr c, .elsewhere ; don't do this
+```
+
+## Labels
+
+```asm
+; ROM Labels
+PascalCase: ; label
```

Comment:
Heh ^^

Hmm, an alternative would be to write 

```asm
; Rom labels
Label:
GlobalLabel::
.localLabel
…
```

but it seems less clear to me, because we're mixing the casing and the purpose.

I think I'd left this part as-this. Is that ok to you?

Reply:
I removed the "no-inline comments" rule anyway, so the case is closed ^^